src/components/RepoList.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div id="repoList">
+    <h1>{{ userName }}</h1>
+    <div id="totalCount">
+      <p>{{ totalRepoCount }} repositories</p>
+      <p>|</p>
+      <p>{{ totalRepoStarCount }} stars</p>
+    </div>
+    <hr>
+    <p v-if="isEmptyRepoList">
```

Comment:
```suggestion
    <p v-if="!totalRepoCount">
```

사소한건데  이렇게 바꾸면 `isEmptyRepoList`를 줄일 수 있을 것 같아요! ㅎㅎ

Reply:
오 감사합니다!